roles/vdm/tasks/postgres/postgres.yaml

```diff
@@ -5,6 +5,19 @@
     - uninstall
     - update
 
+- name: fail - if cds-postgres is external
+  ansible.builtin.fail:
+    msg: "CDS postgres instance must be internal server. To use CDS postgres, please reconfigure your infrastructure to set all your postgres servers internal"
```

Comment:
We might also want to put the link to the SAS documentation stating this limitation : https://go.documentation.sas.com/doc/en/sasadmincdc/v_032/itopssr/p05lfgkwib3zxbn1t6nyihexp12n.htm#n03wzanutmc6gon1val5fykas9aa

Reply:
CONFIG-VARS.md has the link mention the details. I added the link you mentioned for additional information on CDS limitation.